test/integration/framework.go

```diff
@@ -106,7 +107,7 @@ func getFreshApiserverAndClient(t *testing.T, storageTypeStr string) (servicecat
 	return clientset, shutdown
 }
 
-func waitForApiserverUp(serverIP string, stopCh <-chan struct{}) error {
+func waitForApiserverUp(insecureAddr string, stopCh <-chan struct{}) error {
```

Comment:
It could be any url if certs in the k8s world weren't nearly always self-signed. Since I've removed the transport setup that was ignoring cert warnings, there's now an implication here that the url passed in is non-tls. (As for why I removed tls-- these tests are resource hogs and take a while to run. Any small bit of reduced overhead seemed like a win to me.)